pandas/core/series.py

```diff
@@ -4015,6 +4015,13 @@ def _reduce(
         If we have an ndarray as a value, then simply perform the operation,
         otherwise delegate to the object.
         """
+        if name == "var" and is_timedelta64_dtype(self.dtype):
```

Comment:
Should this be `name not in {'mean', 'std'}` so we also block out skew and kurt?

Reply:
I think that gets caught elsewhere